inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceServiceImpl.java

```diff
@@ -87,8 +88,14 @@ public Integer save(SourceRequest request, String operator) {
     }
 
     @Override
-    public SourceResponse get(Integer id, String sourceType) {
-        StreamSourceOperation operation = operationFactory.getInstance(SourceType.forType(sourceType));
+    public SourceResponse get(Integer id) {
+        Preconditions.checkNotNull(id, "source id is empty");
+        StreamSourceEntity entity = sourceMapper.selectById(id);
```

Comment:
There is no need to judge whether it exists or not, because the next `operation.getById()` will change it.